anycode-typescript/azure-pipelines.yml

```diff
@@ -28,7 +28,7 @@ extends:
   parameters:
     buildSteps:
       - script: |
-          env
+          export npm_config_build_from_source=true
```

Comment:
Sadly I cannot move this to `vscode-engineering`, I rely on `task.setvariable` to create env variables https://github.com/microsoft/vscode-engineering/blob/a0284a29ad0c2ba550407c11d9c35f8d85927ee1/azure-pipelines/common/steps/setup-linux-toolchains.yml#L190-L194 but it doesn't preserve case and causes trouble here when parsing https://github.com/prebuild/node-gyp-build/blob/f891aa5f1033d1a3f043398157802098898a86b0/bin.js#L61. 

@lszomoru is there a way to create environment variables from a different task that is available for all tasks in the same job other than using task.setvariable ?